integration-tests/src/test/kotlin/com/google/devtools/ksp/test/PlaygroundIT.kt

```diff
@@ -184,6 +201,13 @@ class PlaygroundIT {
         val gradleProperties = File(project.root, "gradle.properties")
         gradleProperties.appendText("\nkotlin.useK2=true")
```

Comment:
So... this test is just same as `testFirPreview` above, except for passing this gradle property? This might be out of scope of this change, but it'd be good to avoid repetition (of all the other stuff, except for this property setting).

Reply:
It is intentional, this is a deprecated property and we want to ensure the combination of new and deprecated flags won't break KSP (because KSP also reads Kotlin compile flags). This test should be removed once the deprecation becomes an error.